jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -735,13 +750,14 @@ def _update_stretch_histogram(self, msg={}):
                 for attr in ('x_min', 'x_max', 'y_min', 'y_max'):
                     vs_old.remove_callback(attr, self._update_stretch_histogram)
 
-        if self.layer_multiselect:
-            data = self.layer.selected_obj[0][0].layer
-        elif len(self.layer.selected_obj):
-            data = self.layer.selected_obj[0].layer
-        else:
+        if not len(self.layer.selected_obj):
             # skip further updates if no data are available:
             return
+        if isinstance(self.layer.selected_obj[0], list):
+            # multiselect case (but we won't check multiselect since the selection can lag behind)
+            data = self.layer.selected_obj[0][0].layer
+        else:
+            data = self.layer.selected_obj[0].layer

```

Comment:
this just fixes a weird bug I ran into where changing from multi -> single select which then applies the first selection as default triggered a length error here since `self.layer_multiselect` was temporarily out-of-date with the selection.  I can split into a separate PR or changelog entry if anyone feels necessary.